### ISSUE TITLE
[FR] fix cursor navigation in VO/Safari

### ISSUE THREAD

**Issue by thierryk** [tabpanelwidget/tabpanelwidget#9]
In Safari, cursor navigation (VO+arrow) is broken because of the [lack of support for `aria-owns` in Webkit](https://github.com/mdn/content/pull/3235#issuecomment-910921390).  

In that browser, using VO+arrow does not move the cursor from tab to tab, the cursor goes to the open panel instead. Fixing this would seriously reduce the impact of the Webkit bug on VO/Safari users.  

Note that fixing cursor navigation will not fix other issues related to the lack of support in Webkit for `aria-owns`, for example, there is no numbering of tabs in VO/Safari...
